.github/workflows/pr.yml

```diff
@@ -19,36 +19,34 @@ env:
   CTIA_HOOK_ES_INDEXNAME: elasticsearch
   CTIA_HOOK_REDIS_URI: redis://127.0.0.1:6379
   # number of test-matrix builds
-  # (9 jobs for test-matrix + 1 for shellcheck) + 1 for fan-in
+  # 10 jobs + 1 for fan-in
   # allows 2 commits to build simultaneously, since one build
   # will take at most 10 builds at any one time
-  CTIA_NSPLITS: 9
+  CTIA_NSPLITS: 10
   SHELLCHECK_VERSION: v0.7.1 # "stable" or "v0.4.7", or "latest"
-  # shellcheck is preinstalled, use this to override shellcheck for our scripts
-  SHELLCHECK_COMMAND: ~/bin/shellcheck
 
 jobs:
-  shellcheck:
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - name: Install shellcheck
-        run: |
-          (set -x && mkdir -p "${BIN_DIR}")
-          (set -x && wget -qO- "https://github.com/koalaman/shellcheck/releases/download/${SHELLCHECK_VERSION}/shellcheck-${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" | tar -xJv)
-          (set -x && cp "shellcheck-${SHELLCHECK_VERSION}/shellcheck" "${BIN_DIR}/shellcheck")
-          (set -x && $SHELLCHECK_COMMAND --version)
-      - name: Run shellcheck
-        run: ./scripts/shellcheck-build.sh
   test-matrix:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        this-split: [0,1,2,3,4,5,6,7,8] # (range ${CTIA_NSPLITS})
+        this-split: [0,1,2,3,4,5,6,7,8,9] # (range ${CTIA_NSPLITS})
     env:
       CTIA_THIS_SPLIT: ${{ matrix.this-split }}
     steps:
       - uses: actions/checkout@v2
+      - name: ES5 setup
```

Comment:
Moving docker start as early as possible in the build seems a bit faster.

Reply:
:-)
```suggestion
      - name: ES setup
```